src/Microsoft.AspNet.Razor/Generator/GeneratedTagHelperContext.cs

```diff
@@ -35,6 +35,8 @@ public GeneratedTagHelperContext()
             RunnerTypeName = "TagHelperRunner";
             ScopeManagerTypeName = "TagHelperScopeManager";
             ExecutionContextTypeName = "TagHelperExecutionContext";
+            HtmlEncoderPropertyName = "HtmlEncoder";
+            HtmlHelperPropertyName = "Html";
```

Comment:
nit: don't really need two properties and this limits what `MvcRazorHost` can override.  just `HtmlEncoderPropertyName` with a default of `Html.HtmlEncoder` should be fine unless @NTaylorMullen objects.